### PR TITLE
[MIG] iot_amqp_oca: Migration to 13.0

### PR DESCRIPTION
**Pull request by olgamarcocb**
Standard Migration

Change of module name

Depends on #27

### CODE REVIEW

File: iot_amqp_oca/tests/__init__.py [hunk 1]
```diff
@@ -1,4 +1 @@
 from . import test_amqp
-from odoo.addons.iot_output.tests import test_iot
```
Comment (etobella) [inline iot_amqp_oca/tests/__init__.py]: Why did you remove that?

That made some sense, but maybe you should need to do 
`from odoo.addons.iot_output_oca import tests`

Comment (olgamarcocb) [reply iot_amqp_oca/tests/__init__.py]: Added :wink:

**Review by etobella — APPROVED** [review]

**Comment by etobella** [comment]
Can you rebase it please?
